.github/workflows/cypress-token-e2e.yml

```diff
@@ -10,13 +10,15 @@ on:
         required: false
         type: string
         default: /home/runner/go/bin
+      skip-cache:
+        required: false
+        type: string
       tags:
         required: false
         type: string
 
 jobs:
   token-e2e:
-    name: Run Cypress tests - PR
     runs-on: self-hosted
     timeout-minutes: 30
```

Comment:
Is the 30 minute timeout including the night run tests? 30 minutes is cutting it a bit too close, I think 60 minutes would be safer

Reply:
Just increased it :)